x/ibc/07-tendermint/types/errors.go

```diff
@@ -5,7 +5,7 @@ import (
 )
 
 const (
-	SubModuleName = "tendermint"
+	SubModuleName = "ibc-tendermint-client"
```

Comment:
we shouldn't add the ibc- prefix bc commands would turn into `... ibc ibc-tendermint-client ...`. Also if we are adding the client suffix we should add it to the other existing clients too.
```suggestion
	SubModuleName = "tendermint-client"
```